electricitymap/contrib/config/constants.py

```diff
@@ -13,11 +13,6 @@
     "dewpoint",
     "precipitation",
 ]
-OTHER_FORECAST_KEYS = [
-    "price",
-    "production",
-    "consumption",
-]
```

Comment:
Why are these in contrib to start with? 👀

Reply:
I think it's legacy that we copied to the monorepo later but never deleted this 🙈